### PR TITLE
Build Deck Use Case Basic Implementations

### PR DESCRIPTION
**Pull request by stanleypangg**

### CODE REVIEW

File: src/use_cases/build_deck/BuildDeckInputData.java
Comment (ThomasKeough) [inline src/use_cases/build_deck/BuildDeckInputData.java:1]: i think the input data should pass the cards that will become part of the deck to the interactor. then, the interactor will build the deck. this means that the deck doesn't need to be included in the input data, but the 6 cards should be.
^^ this design makes more sense to me but your plan might work better. lmk what u think

File: src/use_cases/build_deck/BuildDeckInteractor.java
Comment (ThomasKeough) [inline src/use_cases/build_deck/BuildDeckInteractor.java:1]: this is where the cards should be passed to the deck initializer

**Review by ThomasKeough — CHANGES_REQUESTED** [review]
concerned about some CA specifics – i think the cards need to be passed to the interactor in the input data object. see my comments and lmk what u think

**Comment by stanleypangg** [comment]
> concerned about some CA specifics – i think the cards need to be passed to the interactor in the input data object. see my comments and lmk what u think

o true, im looking at week5git and realized i did the CA wrong, thanks for your input, i'll change it!

**Review by ThomasKeough — APPROVED** [review]
changes look great

**Pull request merged**